src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # base image
-FROM node:14.17.3-alpine3.13 AS generate-runtime-app
+FROM node:14.17.4-alpine3.13 AS generate-runtime-app
```

Comment:
This whole file kan be deleted, its the same as `src/Altinn.Apps/AppTemplates/react/altinn-app-frontend/Dockerfile` and not in use.